docs/DeveloperGuide.md

```diff
@@ -451,52 +463,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 2.
 
 
-**Use case: Deleting a single resident**
+**Use case: UC 7. Deleting a single resident**
 
 **MSS**
-1. Resident moves out of RC4.
-2. User uses the delete command to remove the resident from RC4HDB.
+1. Resident moves out of [**RC4**](#glossary).
+2. User deletes the resident.
 3. RC4HDB removes the corresponding resident from the database.
 4. RC4HDB displays the details of that resident that has been deleted.
 
     Use case ends.
 
-**Extension**
+**Extensions**
 
-* 2a. User enters an invalid index.
+* 2a. User enters an invalid input.
 
     * 2a1. RC4HDB shows an error message.
 
       Use case resumes at step 2.
 
 
-* 3a. RC4HDB cannot find any resident matching the user input.
+* 2b. RC4HDB cannot find any resident matching the user input.
```

Comment:
Initially we had the find command before the delete command. Now that we don't, I think we should remove this extension. Because, this extension would not occur, any errors would be covered by 2a.